restapi/src/main/java/org/cristalise/restapi/ItemRoot.java

```diff
@@ -447,6 +447,7 @@ public String requestTransition(    String      postData,
     @Path("{binaryUploadPath: .*}")
     public String requestBinaryTransition(  String      postData,
             @FormDataParam ("file")         InputStream file,
+            @FormDataParam("outcome") String outcome,

```

Comment:
I do not see the reason the add the outcome parameter, tha actual outcome is should be provided in postData.

Reply:
`postData` and `FormData` is can not be used at the same time